src/updater.js

```diff
@@ -0,0 +1,18 @@
+import merge from './utils/merge';
+import Template from './template';
+
+export default class Updater {
+  constructor(component) {
+    this.component = component;
+  }
+
+  updateConfig(config) {
+    this.component.config = merge(this.component.config, config.options);
```

Comment:
updateConfig isn't intended for use by end users, just the admin, so I took them out.